test/parallel/test-util-format-shared-arraybuffer.js

```diff
@@ -0,0 +1,11 @@
+// Flags: --harmony_sharedarraybuffer
+
+'use strict';
+require('../common');
+const assert = require('assert');
+const util = require('util');
+
+// Pacify eslint.
+const SharedArrayBuffer = global.SharedArrayBuffer;
```

Comment:
I don't know if there's a preference, but we can use eslints `global` comment to pacify too
```
/* global SharedArrayBuffer */
```